assets/stylesheets/common/common.scss

```diff
@@ -1411,3 +1411,10 @@ body.has-full-page-chat {
     line-height: var(--font-up-1);
   }
 }
+
+.timeline-footer-controls {
+  .topic-chat-button {
+    margin: 0.5em 0 0 0;
+    width: 60%;
```

Comment:
The width was required because the button was not occupying complete space and was looking odd under the timeline buttons. I have now replaced width with `display: inline` which achieves the purpose without setting specific width.